client/src/app/site/projector/components/presentation-control/presentation-control.component.html

```diff
@@ -1,40 +1,40 @@
-<mat-expansion-panel *ngIf="elements.length">
+<mat-expansion-panel *ngIf="(projections | async)?.length">
     <mat-expansion-panel-header>
         <span>{{ 'Media file' | translate }}</span>
     </mat-expansion-panel-header>
-    <div *ngFor="let element of elements">
-        <i>{{ getMediafile(element).getTitle() }}</i>
-        <div *ngIf="getMediafile(element).isImage()">
-            <button type="button" *ngIf="!element.fullscreen" mat-icon-button (click)="fullscreen(element)">
+    <div *ngFor="let projection of projections | async">
```

Comment:
@tsiegleauq IDK why this doesn't work. Can you have a look? The async pipe in line 1 is working -> this media file extension is only shown if a mediafile is projected. Also the console statement in the ts file (line 45) is working, but I do not get any content here.